spec/jobs/resolution_proofing_job_spec.rb

```diff
@@ -46,6 +46,32 @@
       )
     end
 
+    context 'ssn_is_unique attribute' do
+      context 'when the SSN is unique' do
+        it 'sets ssn_is_unique: true on the result' do
+          stub_vendor_requests
+          perform
+
+          result = document_capture_session.load_proofing_result[:result]
+          expect(result[:ssn_is_unique]).to be_truthy
+        end
+      end
+
+      context 'when the SSN is not unique' do
+        before do
+          allow_any_instance_of(Idv::DuplicateSsnFinder).to receive(:ssn_is_unique?).and_return false
```

Comment:
Realistically I think the important thing here is just that `ssn_is_unique` is stored on the result at all, but as long as we're testing it, we might as well test that it's accurate?

Reply:
I feel like the best test would just create another profile with the same SSN as the proofing data so we dont ahve to stub?

Reply:
I had actually started down that path, but found that accessing SSN and creating a new one was a lot more involved than I expected. It seems like the existing user has no profile.

I'm all for doing this right, but am I overlooking a silly helper for this, or should I buckle up and spend a bit updating stuff?

Reply:
We pass in a fixed constant that all comes from `MOCK_IDV_APPLICANT_WITH_SSN` that has a static SSN, so all specs should be using the same SSN, right? So we could

```
before do
  create(:profile, pii: MOCK_IDV_APPLICANT_WITH_SSN)
end
```
and it should set the `ssn_fingerprint` column correctly

Reply:
@zachmargolis D'oh! I was making this way harder than it had to be. This is exactly what I needed. 🥳